ddtrace/tracer/span.go

```diff
@@ -114,6 +114,13 @@ func (s *span) SetTag(key string, value interface{}) {
 	s.Meta[key] = fmt.Sprint(value)
 }
 
+// stackTracer structs implement methods capable of defining a specific StackTrace that should be used.
+// github.com/pkg/errors conforms to it.
+type stackTracer interface {
+	Error() string
+	StackTrace() []interface{}
```

Comment:
```suggestion
	StackTrace() errors.StackTrace
```

This is what Sentry uses, which will give support for `pkg/errors` errors. In Go interfaces will only match on the _exact_ type, so a type will only confirm to the `stackTracer` interface if it has a `StackTrace() []interface{}` method, not if it has a `StackTrace() []<anything_else>` method like you would expect in other languages.